React/my-app/src/components/Book/index.js

```diff
@@ -6,7 +6,7 @@ import { Link } from "react-router-dom";
 const Book = ({ id, imageUrl, title, author }) => (
   <Link to={`/books/${id}`}>
     <div className="book_info">
-      <img className="cover" src={imageUrl || defaultImg} />
+      <img alt={id} className="cover" src={imageUrl || defaultImg} />
```

Comment:
Don't use id, alt should represent the image.
Use the book title or an empty string.